File: sdkwork-app-sdk-csharp/Models/FaqVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class FaqVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? Id { get; set; }
        public string? Question { get; set; }
        public string? CategoryId { get; set; }
        public string? CategoryName { get; set; }
        public int? ViewCount { get; set; }
        public int? HelpfulCount { get; set; }
    }
}

File: sdkwork-app-sdk-go/types/faq_vo.go
package types

// FAQ信息
type FaqVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id string `json:"id"`
	Question string `json:"question"`
	CategoryId string `json:"categoryId"`
	CategoryName string `json:"categoryName"`
	ViewCount int `json:"viewCount"`
	HelpfulCount int `json:"helpfulCount"`
}
